internal/consensus/state.go

```diff
@@ -1377,27 +1389,37 @@ func (cs *State) defaultDoPrevote(height int64, round int32) {
 		return
 	}
 
+	pbtsEnabled := cs.isPBTSEnabled(height)
+	if pbtsEnabled && !cs.Proposal.Timestamp.Equal(cs.ProposalBlock.Header.Time) {
+		logger.Debug("prevote step: proposal timestamp does not match block time; prevoting nil")
```

Comment:
would be good to show the two values in the log message

Reply:
Added.